README.md

```diff
@@ -9,29 +9,71 @@ This project enables Google Batch processing of Illumina DRAGEN pipelines on Evi
 *Note: Illumina sample files are ~90GB*
 
 ```bash
+#!/bin/bash
+
 # Google Cloud Storage bucket for testing
-BUCKET_NAME=
+BUCKET_NAME=<GCS bucket name>
 # Google Cloud project
-PROJECT=
-# get sample files
-mkdir batch-test && cd batch-test
-wget https://ilmn-dragen-giab-samples.s3.amazonaws.com/WGS/precisionFDA_v2_HG002/HG002.novaseq.pcr-free.35x.R1.fastq.gz
-wget https://ilmn-dragen-giab-samples.s3.amazonaws.com/WGS/precisionFDA_v2_HG002/HG002.novaseq.pcr-free.35x.R2.fastq.gz
-mkdir v8 && cd v8
-wget https://ilmn-dragen-giab-samples.s3.amazonaws.com/Hashtable/hg38_altaware-cnv-graph-anchored.v8.tar
-tar -xf hg38_altaware-cnv-graph-anchored.v8.tar && rm hg38_altaware-cnv-graph-anchored.v8.tar && cd ..
+PROJECT=<GCP Project name>
+
 # create storage bucket for test
 gcloud storage buckets --project $PROJECT create gs://$BUCKET_NAME
+
+#method 1 - use storage trasnfer service
```

Comment:
typo: _#method 1 - use storage **trasnfer** service_

Reply:
I fixed in the my fork repo. Please review. Thank you!